src/containers/profile-page/components/desktop/ProfilePage.tsx

```diff
@@ -239,6 +247,20 @@ const ProfilePage = ({
     record(make(Name.TRACK_UPLOAD_OPEN, { source: 'profile' }))
   }, [goToRoute, record])
 
+  const { tierNumber } = useSelectTierInfo(userId ?? 0)
```

Comment:
I feel like this shouldn't default to 0 when not defined?

Reply:
followed an example from the codebase `Tiers.tsx lines 199 and 200`. I believe 0 will just end up giving you '0' for balance anyway. but yeah I agree it's not ideal, can circle back on it